modules/openy_features/openy_location/modules/openy_loc_branch/config/install/core.entity_form_display.node.branch.default.yml

```diff
@@ -117,6 +118,19 @@ third_party_settings:
         description: ''
         required_fields: true
       label: 'Bottom Area'
+    group_amenities:
+      children:
+        - field_location_amenities
+      parent_name: ''
+      weight: 8
+      format_type: tab
+      format_settings:
+        id: ''
+        classes: ''
+        formatter: open
+        description: ''
```

Comment:
Please provide a description, that will provide information about Amenities to the end user.